test/index.test.ts

```diff
@@ -55,6 +62,15 @@ describe('system test', () => {
     scope.done();
   });
 
+  it('should use GCE_METADATA_IP if available', async () => {
+    process.env.GCE_METADATA_IP = '127.0.0.1:8080';
+    const scope = nock(`http://${process.env.GCE_METADATA_IP}`)
+      .get(`${PATH}/${TYPE}/${PROPERTY}`, undefined, HEADERS)
+      .reply(200, {}, HEADERS);
+    await gcp.instance(PROPERTY);
+    scope.done();
```

Comment:
as a final step we should `delete process.env.GCE_METADATA_IP`, so that we're not leaking the environment variable into the next test.